src/features/counterfactual/ActivateAccountButton.tsx

```diff
@@ -24,24 +25,28 @@ const ActivateAccountButton = () => {
   return (
     <Tooltip title={isProcessing ? 'The safe activation is already in process' : undefined}>
       <span>
-        <Button
-          data-testid="activate-account-btn"
-          variant="contained"
-          size="small"
-          onClick={activateAccount}
-          disabled={isProcessing}
-        >
-          {isProcessing ? (
-            <>
-              <Typography variant="body2" component="span" mr={1}>
-                Processing
-              </Typography>
-              <CircularProgress size={16} />
-            </>
-          ) : (
-            'Activate now'
+        <CheckWallet allowNonOwner>
+          {(isOk) => (
+            <Button
+              data-testid="activate-account-btn"
+              variant="contained"
+              size="small"
+              onClick={activateAccount}
```

Comment:
```suggestion
              onClick={activateAccount}
                sx={{ minHeight: '40px' }}
```

So that both buttons have the same size on the dashboard